docs/integrations/data-ingestion/clickpipes/postgres/source/alloydb.md

```diff
@@ -0,0 +1,151 @@
+   CREATE USER clickpipes_user PASSWORD 'some-password';
+   ```
+
+2. Grant the dedicated user permissions on the schema(s) you want to replicate.
+   
+    ```sql
+    GRANT USAGE ON SCHEMA "public" TO clickpipes_user;
+    GRANT SELECT ON ALL TABLES IN SCHEMA "public" TO clickpipes_user;
+    ALTER DEFAULT PRIVILEGES IN SCHEMA "public" GRANT SELECT ON TABLES TO clickpipes_user;
+    ```
+
+   The example above shows permissions for the `public` schema. Repeat the sequence of commands for each schema you want to replicate using ClickPipes.
+
+3. Grant the dedicated user permissions to manage replication:
+
+   ```sql
+   ALTER ROLE clickpipes_user REPLICATION;
+   ```
+
+4. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate. We strongly recommend only including the tables you need in the publication to avoid performance overhead.
+
+   :::warning
+   All tables included in the publication must either have a **primary key** defined _or_ have its **replica identity** configured to `FULL`. See the [Postgres FAQs](../faq.md#how-should-i-scope-my-publications-when-setting-up-replication-how-should-i-scope-my-publications-when-setting-up-replication) for guidance on scoping.
+   :::
+
+   - To create a publication for specific tables:
+
+      ```sql
+      CREATE PUBLICATION clickpipes FOR TABLE table_to_replicate, table_to_replicate2;
+      ```
+
+   - To create a publication for all tables in a specific schema:
+
+      ```sql
+      CREATE PUBLICATION clickpipes FOR TABLES IN SCHEMA "public";
+      ```
+
+   The `clickpipes` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
+
+## Configure network security {#configure-network-security}
+
+:::note
+ClickPipes does not support Private Service Connect (PSC) connections. If you do not allow public access to your AlloyDB instance, you can [use an SSH tunnel](#configure-network-security) to connect securely. PSC will be supported in the future.
+:::
+
+Next, you must allow connections to your AlloyDB instance from ClickPipes.
+
+<Tabs groupId="network-configuration">
+<TabItem value="public-ip" label="Allow ClickPipes IPs">
+
+1. In the Google Cloud Console, navigate to the AlloyDB [Clusters](https://console.cloud.google.com/alloydb/clusters) page. Select your primary instance to open the **Overview** page.
+
+2. Scroll down to **Instances in your cluster** and click **Edit primary**.
+
+3. Check the **Enable Public IP** checkbox to allow connections to the instance over the public internet. Under **Authorized external networks**, enter the [list of ClickPipes static IP addresses](../../../clickpipes/index.md#list-of-static-ips-list-of-static-ips) for the region your service is deployed in.
+
+   <Image img={configure_network_security} alt="Configure networking for public access with an IP allowlist" size="lg" border/>
```

Comment:
(nit, up to you): Should the list of IPs on the screenshot be visibly truncated or have a label on top that both covers it and directs to use the list from the other page? Worry is that someone inattentive retypes it from the pic and gets surprised it doesn't work